src/main/java/seedu/address/model/note/Note.java

```diff
@@ -1,28 +1,38 @@
 package seedu.address.model.note;
 
+import static java.util.Objects.requireNonNull;
+
 public class Note {
+
     // TODO: Add implementation
-//    private final String noteContents;
-//
-//    public Note(String content) {
-//        requireNonNull(content)
-//        noteContents = content;
-//    }
-//    @Override
-//    public String toString() {
-//        return noteContents;
-//    }
-//    @Override
-//    public boolean equals(Object other) {
-//        if (other == this) {
-//            return true;
-//        }
-//        if (!(other instanceof Note)) {
-//            return false;
-//        }
-//    }
-//    @Override
-//    public int hashCode() {
-//
-//    }
+    public static final String MESSAGE_CONSTRAINTS = "message contraints for note";
```

Comment:
Might want to add a TODO here

Reply:
I think this should be done by @rtshkmr since he is going to implement `Note` fully.